### PR TITLE
adding logging on access node for execute script calls

### PR DESCRIPTION
**Pull request by vishalchangrani**

### CODE REVIEW

File: engine/access/rpc/backend/backend_scripts.go [hunk 1]
```diff
@@ -112,16 +121,17 @@ func (b *backendScripts) executeScriptOnExecutionNode(
...
-				Msg("Successfully executed script")
+				Msg("successfully executed script")
 			return result, nil
 		}
 		errors = multierror.Append(errors, err)
 	}
-	return nil, errors.ErrorOrNil()
+
+	err = errors.ErrorOrNil()
+	logger.Err(err).Msg("failed to execute script")
```
Comment (Kay-Zee) [inline engine/access/rpc/backend/backend_scripts.go:133]: Does this error include all the exe nodes we tried? if not should we add that to this log message?

Comment (vishalchangrani) [reply engine/access/rpc/backend/backend_scripts.go:133]: yeah it does include the exe node name as part of the message (https://github.com/onflow/flow-go/pull/452/files#diff-d4f96095c328265258f0b505e475292ed7d9460a4591033eae31271798ad74caR140)

File: engine/access/rpc/backend/backend_scripts.go [hunk 2]
```diff
@@ -86,24 +86,33 @@ func (b *backendScripts) executeScriptOnExecutionNode(
 		Arguments: arguments,
 	}
 
+	logger := b.log.With().
+		Hex("block_id", blockID[:]).
+		Str("script", string(script)).
```
Comment (ramtinms) [inline engine/access/rpc/backend/backend_scripts.go:91]: you might also capture `arguments` as well

**Comment by vishalchangrani** [comment]
logging of script has been already included on master. Closing this PR.

**Pull request closed without merging**